app/src/main/java/org/fossasia/openevent/app/event/attendees/AttendeesPresenter.java

```diff
@@ -0,0 +1,68 @@
+package org.fossasia.openevent.app.event.attendees;
+
+import android.content.Context;
+
+import org.fossasia.openevent.app.data.EventDataRepository;
+import org.fossasia.openevent.app.data.UtilModel;
+import org.fossasia.openevent.app.data.contract.IEventDataRepository;
+import org.fossasia.openevent.app.data.contract.IUtilModel;
+import org.fossasia.openevent.app.data.models.Attendee;
+import org.fossasia.openevent.app.event.attendees.contract.IAttendeesPresenter;
+import org.fossasia.openevent.app.event.attendees.contract.IAttendeesView;
+
+public class AttendeesPresenter implements IAttendeesPresenter {
+
+    IEventDataRepository eventDataRepository;
+    IAttendeesView attendeesView;
+    Long eventId;
+
+    public AttendeesPresenter(IAttendeesView attendeesView, Context context, Long eventId) {
+        this.attendeesView = attendeesView;
+        eventDataRepository = new EventDataRepository(new UtilModel(context));
+        this.eventId = eventId;
+    }
+
+    @Override
+    public void attach() {
+        loadAttendees(true);
+    }
+
+    @Override
+    public void dettach() {
+        attendeesView = null;
+    }
+
+    @Override
+    public void loadAttendees(boolean forceReload) {
+        attendeesView.showProgressBar(true);
+        eventDataRepository.getAttendees(eventId, forceReload)
+            .subscribe(attendees -> {
+                attendeesView.loadAttendeesIntoList(attendees);
```

Comment:
View doesn't know what is list, view should just be provided with just the data and said to show it. We are definitely hiding and showing attendees, because in case there are no attendees, we will hide attendees and show empty view in future